distributed/deploy/utils_test.py

```diff
@@ -1,38 +0,0 @@
-from __future__ import annotations
-
-from typing import Any
-
-import pytest
-
-from distributed.client import Client
-
-
-class ClusterTest:
-    Cluster = None
-    kwargs: dict[str, Any] = {}
-
-    def setUp(self):
-        self.cluster = self.Cluster(2, scheduler_port=0, **self.kwargs)
-        self.client = Client(self.cluster.scheduler_address)
-
-    def tearDown(self):
-        self.client.close()
-        self.cluster.close()
-
-    @pytest.mark.xfail()
```

Comment:
this test failed because `self.Cluster(2, ` made a `LocalCluster(name=2, n_workers=nprocesses_nthreads()`